resources/resources/software/aws/toolkits/resources/MessagesBundle.properties

```diff
@@ -604,7 +604,7 @@ codewhisperer.codescan.build_artifacts_not_found=<html>Cannot find build artifac
 codewhisperer.codescan.cannot_read_file=Error while reading file {0}
 codewhisperer.codescan.cwe_label=Common Weakness Enumeration (CWE)
 codewhisperer.codescan.detector_library_label=Detector library
-codewhisperer.codescan.file_ext_not_supported=File extension {0} is not supported for CodeWhisperer Security Scan feature. Please try again with a valid file format - java, python, javascript, typescript, csharp, yaml, json, tf, hcl.
+codewhisperer.codescan.file_ext_not_supported=File extension {0} is not supported for CodeWhisperer Security Scan feature. Please try again with a valid file format - java, python, javascript, typescript, csharp, yaml, json, tf, hcl, ruby, go.
```

Comment:
should probably refer to these by their proper name (e.g. Java, C#, Terraform)

Reply:
Can we keep only this `File extension {0} is not supported for CodeWhisperer Security Scan feature.` because in VSCode we don't mention all the valid file formats ?